.github/workflows/test-chart.yaml

```diff
@@ -394,3 +394,92 @@ jobs:
         if: failure() && matrix.setup-postgresql-args
         run: |
           kubectl logs statefulset/postgresql
+
+  test-gateway-ingress:
+    # The main test configures autohttps, but this conflicts with testing
+    # ingress/gateway which assume they can directly connect to proxy-public
```

Comment:
In theory it should be possible to have autohttps working, but that would only be used for internal connections between the traefik ingress/gateway controller and the autohttps/proxy pod which effectively means it's no use, since the certificate is designed for external connections.

The certificate for external connections has to be managed by the gateway or ingress controller, not autohttps.
https://gateway-api.sigs.k8s.io/guides/tls/

Reply:
Rereading https://gateway-api.sigs.k8s.io/guides/tls/#tls-configuration there's a `Passthrough` mode, but this is _Experimental_ and I don't see what benefit it offers for autohttps.